src/components/ToggleColorModeButton.tsx

```diff
@@ -0,0 +1,17 @@
+import { Button, useColorMode } from '@chakra-ui/react'
+
+const ToggleColorModeButton = () => {
+  const { colorMode, toggleColorMode } = useColorMode()
+  const isLightMode = colorMode === 'light'
+
+  return (
+    <Button
+      onClick={toggleColorMode}
+      colorScheme={isLightMode ? 'primary' : 'secondary'}
+    >
+      Toggle {isLightMode ? 'Dark' : 'Light'}
+    </Button>
```

Comment:
This button is ugly as fuck and doesn't correspond to the designs, but it's just a mechanism to allow you to force light or dark mode. It will be styled properly when releasing it officially to production.

Reply:
I got it working on my end with your detailed instructions. Hooray! 🎉 

That button is _nowhere near_ the final design, I'm glad we hide it from public view... 🤪 😉 

I really like the idea to put this feature behind a flag, so that we can work on it "behind the scenes", while still pushing new code to `main` instead of to a `dark_mode_branch`! 😄

Reply:
Always better to push everything to `main` and simplify the workflow. It's simple to hide things like this behind a flag.